modules/gdnative/nativescript/api_generator.cpp

```diff
@@ -292,6 +292,7 @@ List<ClassAPI> generate_c_api_classes() {
 				method_api.has_varargs = method_bind && method_bind->is_vararg();
 
 				// Method flags
+				method_api.is_virtual = false;
```

Comment:
I'm not sure I understand the need for this line. Doesn't `method_api.is_virtual` get set later in the file?

Reply:
[modules/gdnative/nativescript/api_generator.cpp:305]: (error) Uninitialized struct member: method_api.is_virtual

if method_info.flags on line 296 is false we will reassign an undefined value of method_api.is_virtual to method_api.is_virtual on line 306.

Reply:
Nevermind, I forgot C++ isn't guaranteed to give bools a default `false` value. Woops.

You can mark this as resolved.